### PR TITLE
docs(msgbox): update to reflect latest

### LINKED ISSUE #6433: documentation for msgbox needs to be updated.
### LVGL version

master

### What happened?

the documentation overview is incorrect for how to use this widget.

### How to reproduce?

https://docs.lvgl.io/master/widgets/msgbox.html

### PR DESCRIPTION
**Pull request by liamHowatt**
### Description of the feature or fix

Fixes #6433

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

File: examples/widgets/msgbox/index.rst [hunk 1]
```diff
@@ -5,4 +5,7 @@ Simple Message box
 .. lv_example:: widgets/msgbox/lv_example_msgbox_1
   :language: c
 
+.. lv_example:: widgets/msgbox/lv_example_msgbox_2
```
Comment (lhdjply) [inline examples/widgets/msgbox/index.rst:11]: We should add titles to the examples here.

Comment (liamHowatt) [reply examples/widgets/msgbox/index.rst:11]: It seems that no widget doc has that. See any `docs/widgets/*.rst`.

**Comment by lhdjply** [comment]
Screenshot_20240731_233220

File: examples/widgets/msgbox/index.rst (hunk 1 above)
Comment (lhdjply) [reply examples/widgets/msgbox/index.rst:11]: But you changed `examples/widgets/**/index.rst`
Screenshot_20240731_233220

Comment (liamHowatt) [reply examples/widgets/msgbox/index.rst:11]: Ah yes, thanks, great catch. Please forgive me, I was looking at the wrong file.

**Review by lhdjply — APPROVED** [review]

**Review by kisvegabor — APPROVED** [review]
Thank you Liam!

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**